src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -77,18 +77,15 @@ void reshape_to_static(std::shared_ptr<ov::Model> model,
     model->reshape(new_shapes);
 }
 
-void fill_tensor(ov::Tensor tensor, int64_t fill_val) {
+void fill_tensor(ov::Tensor tensor, int64_t fill_val, int32_t offset = 0) {
```

Comment:
Why not `size_t`. Or `ptrdiff_t` if you expect a negative offset, which doesn't make sense for `tensor_data + offset`? They are for pointer arithmetic.

Reply:
Overlooked it, thanks, fixed